rcsb/db/wf/RepoLoadWorkflow.py

```diff
@@ -394,7 +398,7 @@ def getCsmHash(self, pdbid):
         return os.path.join(pdbid[0:2], pdbid[-6:-4], pdbid[-4:-2])
 
     def getContentTypePrefix(self, databaseName):
-        if databaseName == "pdbx_core":
+        if databaseName in ["pdbx_core", "pdbx_ihm"]:
```

Comment:
Would it be better to use `contentType` instead, given that that's supposed to be cover all three cases? I.e., as defined above:
```python
        databaseName = kwargs.get("databaseName")  # 'pdbx_core' or 'pdbx_comp_model_core'
        contentType = kwargs.get("contentType", None)
        contentType = contentType if contentType else databaseName  # 'pdbx_core', 'pdbx_comp_model_core', 'pdbx_ihm'
```